lib/install/validate/validator.go

```diff
@@ -320,20 +324,55 @@ func (v *Validator) target(ctx context.Context, input *data.Data, conf *config.V
 	// check if host is managed by VC
 	v.managedbyVC(ctx)
 
+	// check target with ops credentials
 	u := input.Target.URL
 
+	conf.Username = input.OpsUser
+	conf.Token = input.OpsPassword
+	conf.TargetThumbprint = input.Thumbprint
+
 	// Discard anything other than these URL fields for the target
-	conf.Target = (&url.URL{
+	stripped := &url.URL{
 		Scheme: u.Scheme,
 		Host:   u.Host,
 		Path:   u.Path,
-	}).String()
+	}
+	conf.Target = stripped.String()
 
-	conf.Username = u.User.Username()
-	conf.Token, _ = u.User.Password()
-	conf.TargetThumbprint = input.Thumbprint
+	// validate that the provided operations credentials are valid
+	stripped.Path = "/sdk"
+
+	var soapClient *soap.Client
```

Comment:
Technically no, given our current logic flow in vic-machine.

I prefer to keep the current ignorance however as once we've moved from session to the vmomi gateway structure, validator won't need to see/know the credentials that are being used for validation.